libs/ml/include/ml/layers/scaled_dot_product_attention.hpp

```diff
@@ -0,0 +1,120 @@
+#pragma once
```

Comment:
Please make sure that you didn't break anything that was using old self_attention.hpp .